kani-compiler/src/codegen_cprover_gotoc/codegen/statement.rs

```diff
@@ -364,10 +369,10 @@ impl<'tcx> GotocCtx<'tcx> {
                 }
             }
             _ => unreachable!(
-                "TerminatorKind::Drop but not InstanceDef::DropGlue should be impossible"
+                "TerminatorKind::Drop but not InstanceKind::DropGlue should be impossible"
```

Comment:
We can improve this error message. Let's avoid `InstanceKind::DropGlue` in it because it doesn't exist anymore.